index.html

```diff
@@ -262,59 +290,72 @@ <h2>About me</h2>
+                        </div>
+                        <ul class="skillsUl flexColumn">
+                            <li class="skillsLi inter">JavaScript</li>
+                            <li class="skillsLi inter">Ruby</li>
+                            <li class="skillsLi inter">HTML</li>
+                            <li class="skillsLi inter">CSS</li>
+                        </ul>
                     </div>
-                    <ul class="skillsUl flexColumn">
-                        <li class="skillsLi inter">JavaScript</li>
-                        <li class="skillsLi inter">Ruby</li>
-                        <li class="skillsLi inter">HTML</li>
-                        <li class="skillsLi inter">CSS</li>
-                    </ul>
-                </div>
-                <div class="skillsCont flexColumn">
-                    <div class="sTitle inter">
-                        <h2><img src="style/icon/square.svg" alt="square icon"> Frameworks</h2>
+                    <div class="skillsCont flexColumn">
+                        <div class="sTitle inter">
+                            <h2><img src="style/icon/square.svg" alt="square icon"> Frameworks</h2>
+                        </div>
+                        <ul class="skillsUl">
+                            <li class="skillsLi inter">Bootstrap</li>
+                            <li class="skillsLi inter">Ruby on Rails</li>
+                            <li class="skillsLi inter">RSPec</li>
+                            <li class="skillsLi inter">CapyBara</li>
+                            <li class="skillsLi inter">Selenium</li>
+                        </ul>
                     </div>
-                    <ul class="skillsUl">
-                        <li class="skillsLi inter">Bootstrap</li>
-                        <li class="skillsLi inter">Ruby on Rails</li>
-                        <li class="skillsLi inter">RSPec</li>
-                        <li class="skillsLi inter">CapyBara</li>
-                        <li class="skillsLi inter">Selenium</li>
-                    </ul>
-                </div>
-                <div class="skillsCont flexColumn">
-                    <div class="sTitle inter">
-                        <h2><img src="/style/icon/circle.svg" alt="circle"> Skills</h2>
+                    <div class="skillsCont flexColumn">
+                        <div class="sTitle inter">
+                            <h2><img src="/style/icon/circle.svg" alt="circle"> Skills</h2>
+                        </div>
+                        <ul class="skillsUl">
+                            <li class="skillsLi inter">Codekit</li>
+                            <li class="skillsLi inter">GitHub</li>
+                            <li class="skillsLi inter">CodePen</li>
+                            <li class="skillsLi inter">GitLab</li>
+                            <li class="skillsLi inter">Terminal</li>
+                        </ul>
```

Comment:
- [ ] You have a good eye for design 🤗 but I think you need to check why the background image for this section and the prepended icon to the left of `Skills` are not showing as your displayed in your descriptive pull request message.

![image](https://user-images.githubusercontent.com/99715640/220823865-a9975f86-d7df-4067-8889-ad475d40a557.png)

Reply:
Coorected!